eng/illink.targets

```diff
@@ -9,8 +9,8 @@
 
   <PropertyGroup>
     <TargetsTriggeredByCompilation Condition="'$(DesignTimeBuild)' != 'true'">
-      $(TargetsTriggeredByCompilation);
-      ILLinkTrimAssembly
+      ILLinkTrimAssembly;
+      $(TargetsTriggeredByCompilation)
```

Comment:
I think this is a reasonable change.  Another thing to think about might be to explicitly add this to `ApiCompatDependsOn`